integration-tests/js-compute/fixtures/app/src/geoip.js

```diff
@@ -148,11 +148,29 @@ let geoFields = [
   'utc_offset',
 ];
 
+routes.set('/fastly/getgeolocationforipaddress/bad-ip', async () => {
+  let geo = fastly.getGeolocationForIpAddress('0.0.0.0');
+  if (isRunningLocally()) {
+    strictEqual(
+      geo,
+      null,
+      `fastly.getGeolocationForIpAddress('0.0.0.0') == null`,
+    );
+  } else {
+    strictEqual(geo.as_name, '?');
+  }
+  assertThrows(() => {
+    fastly.getGeolocationForIpAddress('999.999.999.999');
+  }, Error);
+});
+
 routes.set(
   '/fastly/getgeolocationforipaddress/parameter-ipv4-string',
   async () => {
+    let geo = fastly.getGeolocationForIpAddress('1.9.2.1');
```

Comment:
WHOIS shows `1.9.2.1` as part of an allocated network range:

```
inetnum:        1.9.2.0 - 1.9.2.255
netname:        MYREN-TMNET
descr:          Kementerian Pengajian Tinggi Malaysia
country:        MY
admin-c:        KK753-AP
tech-c:         KK753-AP
abuse-c:        AM2355-AP
status:         ASSIGNED NON-PORTABLE
mnt-by:         MAINT-MY-WEBE
mnt-irt:        IRT-MY-WEBE
last-modified:  2023-06-30T00:01:26Z
source:         APNIC
[ ... ]
```

Is the idea that we expect valid data for this address (so we test the `keys` on the object below)? If so, could we pick an address we control, or one that is well-known, to avoid any potential test flakes in the future? E.g. either a Fastly IP (`151.101.1.1` as the first IP in our main /16) or Google's DNS (`8.8.8.8`) or something?

Reply:
(Should match the assert error message below, and likewise same question for the IPv6 address below, which appears to be in a Softlayer IP range)

Reply:
Good suggestion - I've gone with Fastly's IP here instead.